generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -42,6 +42,21 @@ if (reactive) {
     callBlock = ".block()";
     callListBlock = ".collectList().block()";
 }
+let repositorySave = 'save';
```

Comment:
I think naming it `saveMethod` (or just `save`) would make understanding the templates a little easier:
```
<%= entityInstance %>Repository.<%= saveMethod %>
<%= entityInstance %>Repository.<%= save %>
```

Reply:
Done